### PR TITLE
Update Tizen 5.5 TAU documentation

### PR DESCRIPTION
**Pull request by lslachciak**
Updated TAU 5.5 documentation with links to examples.
Synchronized minor improvements in documentation between 5.0 and 5.5

### CODE REVIEW

**Comment by lslachciak** [comment]
sample stg url: https://docs.stg.tizen.org/staging/938/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Button.htm

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm [hunk 1]
```diff
@@ -24,6 +24,7 @@ <h1>Button</h1>
 >Use example</button>
 
 <p>The following table describes the supported button classes.</p>
+<img src="../../images/button.png" alt="photo of button element in Tizen">
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: Duplicated img. Please remove this change.

Comment (lslachciak) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm]: removed, also in 5.5 documentation

**Comment by lslachciak** [comment]
@mijin85cho could you approve and merge?

**Comment by lslachciak** [comment]
Fixed wearable buttons documentation according to @TomaszLukawskiSam  comment
https://docs.stg.tizen.org/staging/938/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_button.htm

**Comment by lslachciak** [comment]
@mijin85cho couold you review/merge 
this PR was internally approved

**Comment by mijin85cho** [comment]
Hello, @lslachciak.
I have a question based on the https://github.com/Samsung/tizen-docs/pull/797.
It looks like we need to pass the verification test for the links, according to the https://github.com/Samsung/tizen-docs/pull/797.

Don't we need to proceed that process for this PR?

**Comment by lslachciak** [comment]
@mijin85cho we are checking if this PR requires verification or not,

let's hold with merging this PR before decision.

**Comment by singa2000** [comment]
We get the verification PASS.
Would you merge it, please?

**Comment by mijin85cho** [comment]
Same issue with https://github.com/Samsung/tizen-docs/pull/940.

Would you please refer [this comment](https://github.com/Samsung/tizen-docs/pull/940)?

>Question. I can't merge this PR as there are conflicted files.
>Also, it looks like the some fixes in this file are duplicated with #959. Would you please check?

**Comment by lslachciak** [comment]
@mijin85cho @singa2000 I suggest merging https://github.com/Samsung/tizen-docs/pull/959/commits
and closing this PR. It already contains this commit and is up to date

**Comment by mijin85cho** [comment]
> @mijin85cho @singa2000 I suggest merging https://github.com/Samsung/tizen-docs/pull/959/commits
> and closing this PR. It already contains this commit and is up to date

I agree with your suggestion. 
The updates are included in #959, so when the issues of #959 are fixed, I think we can merge #959 and just close #940 and #938.

**Comment by mijin85cho** [comment]
> > @mijin85cho @singa2000 I suggest merging https://github.com/Samsung/tizen-docs/pull/959/commits
> > and closing this PR. It already contains this commit and is up to date
> 
> I agree with your suggestion.
> The updates are included in #959, so when the issues of #959 are fixed, I think we can merge #959 and just close #940 and #938.

https://github.com/Samsung/tizen-docs/pull/959 is merged. So I close this PR as we discussed.

**Pull request closed without merging**